lib/moku.rb

```diff
@@ -47,7 +47,6 @@ def initialize!
           container.register(:system_runner) { Moku::Shell::Basic.new }
         end
         container.register(:remote_runner) {|c| Moku::Shell::SecureRemote.new(c.system_runner) }
-        container.register(:backend_runner) {|c| Moku::CapRunner.new(c.system_runner) }
         container.register(:upload_factory) { Moku::Upload }
```

Comment:
I'm not clear on how this relates. Does it relate? Or is it some unrelated housekeeping?

Reply:
Yes, see cc64209187d3a6601c2454a8698c9f34600f514d